### PR TITLE
Add original documentation

### LINKED ISSUE #99: Add original pdf documentation to the docs
We probably want to include the original documentation for command line running. This may be superseeded but for now we just want to add it to github and link to it from the main sphinx documentation:

https://gitlab.lam.fr/Galaxies/LEPHARE/-/blob/master/doc/LePhare_documentation.pdf?ref_type=heads

### PR DESCRIPTION
**Pull request by raphaelshirley**
So far I have just added the latex files.

Can these be built to html?

Closes #99 #115 

- [ ] My PR includes a link to the issue that I am addressing

## Code Quality
- [ ] I have read the Contribution Guide
- [ ] My code follows the code style of this project
- [ ] My code builds (or compiles) cleanly without any errors or warnings
- [ ] My code contains relevant comments and necessary documentation

### Bug Fix Checklist
- [ ] My fix includes a new test that breaks as a result of the bug (if possible)
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### New Feature Checklist
- [ ] I have added or updated the docstrings associated with my feature using the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover my new feature
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### Documentation Change Checklist
- [ ] Any updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)

### Build/CI Change Checklist
- [ ] If required or optional dependencies have changed (including version numbers), I have updated the README to reflect this
- [ ] If this is a new CI setup, I have added the associated badge to the README

### Other Change Checklist
- [ ] Any new or updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover any changes
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### CODE REVIEW

**Comment by raphaelshirley** [comment]
I realise I branched off from the process branch. I will wait to see what happens with the other branch unless someones has a clear idea how to revert the changes from the source branch.

**Comment by raphaelshirley** [comment]
We probably want to manually check the pandoc conversion and add in the figure etc before merging.

File: docs/index.rst [hunk 1]
```diff
@@ -4,66 +4,28 @@
...
-   that a set of tests will be run prior to completing a local commit. For more
-   information, see the Python Project Template documentation on
-   `pre-commit <https://lincc-ppt.readthedocs.io/en/latest/practices/precommit.html>`_.
-3) Installing ``pandoc`` allows you to verify that automatic rendering of Jupyter notebooks
-   into documentation for ReadTheDocs works as expected. For more information, see
-   the Python Project Template documentation on
-   `Sphinx and Python Notebooks <https://lincc-ppt.readthedocs.io/en/latest/practices/sphinx.html#python-notebooks>`_.
+   This is a work in progress, and should not be used by anyone at this stage. The 
+   current LePHARE code is to be found at `https://gitlab.lam.fr/Galaxies/LEPHARE 
+   <https://gitlab.lam.fr/Galaxies/LEPHARE>`_.
```
Comment (OliviaLynn) [inline docs/index.rst:12]: Do we want to include links to https://github.com/lephare-photoz/lephare and https://github.com/LSSTDESC/rail_lephare?

**Review by johannct — APPROVED** [review]
Let's get it in so that we can start rationalizing the Documentation

**Pull request merged**